web_external/js/views/body/StudyResultsView.js

```diff
@@ -589,10 +587,7 @@ isic.views.StudyResultsView = isic.View.extend({
             this.images.reset(imageModels);
 
             // Populate users collection
-            var userModels = _.map(this.study.get('users'), function (user) {
-                return new girder.models.UserModel(user);
-            });
-            this.users.reset(userModels);
+            this.users.reset(this.study.users());
```

Comment:
`reset()` can't handle a `Collection`, but we could use `this.study.users().models`.